CMake/ClangCompilers.cmake

```diff
@@ -17,9 +17,14 @@ ENDIF()
 # Set clang specfic flags (which we always want)
 ADD_DEFINITIONS( -Drestrict=__restrict__ )
 
-SET(CMAKE_C_FLAGS     "${CMAKE_C_FLAGS} -fomit-frame-pointer -fstrict-aliasing")
-SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fomit-frame-pointer -fstrict-aliasing -D__forceinline=inline")
-SET( HAVE_POSIX_MEMALIGN 0 )    # Clang doesn't support -malign-double
+IF (CMAKE_CXX_COMPILER_VERSION VERSION_GREATER 5.0 )
+  SET( HAVE_POSIX_MEMALIGN 1 )
+  SET( CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -D__forceinline=inline" )
```

Comment:
forceinline is needed in *any* case for clang. Please move it to CMAKE_CXX_FLAGS.

Reply:
My debug works fine without it.

Reply:
Even without `-D__forceinline=''`it works?

Reply:
I prefer a clean solution as I explained above. It should also not inside version check. It is needed on PowerPC Clang 3.8.

Reply:
No it doesn't __forceinline is a compiler specific tag that shouldn't be in a portable codebase. However this sort of intense inlining is also something I wouldn't expect enabled in a debug build.

Reply:
You are right. It is compiler specific. There are two ways to protect it, 1 put macro in source, 2 add option in the compiler configuration. For the moment, 2 is simpler.